src/commands/locks/list.ts

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2022, Nuance, Inc. and its contributors.
+ * All rights reserved.
+ *
+ * This source code is licensed under the Apache-2.0 license found in
+ * the LICENSE file in the root directory of this source tree.
+ */
+
+import chalk from 'chalk'
+import {flags} from '@oclif/command'
+import makeDebug from 'debug'
+
+import {defaultLimit} from '../../utils/constants'
+import * as MixFlags from '../../utils/flags'
+import * as ProjectsAPI from '../../mix/api/projects'
+import {DomainOption} from '../../utils/validations'
+import {MixClient, MixResponse, MixResult,  ProjectsLockListParams} from '../../mix/types'
+import MixCommand from '../../utils/base/mix-command'
+
+const debug = makeDebug('mix:commands:locks:list')
+
+export default class LocksList extends MixCommand {
+  static description = `list project locks
+
+Use this command to get a list of all project locks.
+The list can be constrained using the project, organization
+and/or user flags.
+A project cannot be edited while it is locked.`
+
+  static examples = [
+    'list project locks',
+    '$ mix locks:list -P 249 -O 33 -U 32',
+  ]
+
+  static flags = {
+    json: MixFlags.jsonFlag,
+    project: {
+      ...MixFlags.projectWithDefaultFlag,
+      required: false,
+    },
+    user: MixFlags.userFlag,
+    organization: {
+      ...MixFlags.organizationWithDefaultFlag,
+      required: false,
+    },
+    ...MixFlags.limitOffsetSortFlags,
+    ...MixFlags.tableFlags({except: ['extended', 'sort']}),
+    yaml: MixFlags.yamlFlag,
+  }
+
+  get columns() {
+    debug('get columns()')
+    return {
+      lockId: {header: 'LockId'},
+      projectId: {header: 'ProjectId'},
+      email: {
+        header: 'LockOwnerName',
+        get: (locks: any) => locks.lockOwner.email,
```

Comment:
Let's display the actual name first, so `locks.lockOwner.name` and add a new column right after name to display the email address (LockOwnerEmail will be the column name).